sass/base/_typography.scss

```diff
@@ -0,0 +1,5 @@
+a{

```

Comment:
You may want to add a `:hover` pseudoclass so there is consistent styling when a cursor is placed over the link